repo/packages/hyperdrive/package.py

```diff
@@ -0,0 +1,75 @@
+from spack.package import *
+
+class Hyperdrive(Package, ROCmPackage, CudaPackage):
+    """A preprocessing pipeline for the Murchison Widefield Array"""
+
+    homepage = "https://github.com/MWATelescope/mwa_hyperdrive"
+    git = "https://github.com/MWATelescope/mwa_hyperdrive.git"
+
+    maintainers = ["d3v-null"]
+
+    version("main",  branch="main")
+    version("0.4.1", tag="v0.4.1")
+
+    variant("plotting", default=True, description="Enable plotting subcommands like plot-solutions")
+
+    depends_on("rust@1.64.0:")
+    depends_on("cmake", type="build")
+    # cfitsio > 4 introduces a breaking change, is incompatible with mwalib.
+    # curl is needed because cfitsio does not --disable-curl by default
+    depends_on("cfitsio@3.49")
```

Comment:
Hi @d3v-null , please use the `+reentrant` variant of `cfitsio`, and possibly add the same logic as in `mwalib.py`, lines 124-125.

This is the only package that fails now